src/sentry/lang/native/symbolizer.py

```diff
@@ -79,7 +79,7 @@ def is_sdk_failure(self):
 
     def get_data(self):
         """Returns the event data."""
-        rv = {'message': self.message}
+        rv = {'message': self.message, 'type': self.type}
```

Comment:
Note that this is now also included in the `data` field of reprocessing reports. If we don't want it there, I'd suggest that we remove it again in `report_processing_issue`.